pipelines/reconfigure.yml

```
@@ -219,6 +222,7 @@ jobs:
         vars:
           release_major: "6"
           release_minor: "6.0"
+          latest_release: "5.7.2"
```

```suggestion
          latest_release: "5"
```
might be more future-proof

---

Good call thx!